.travis.yml

```diff
@@ -7,18 +7,23 @@ cache:
 env:
 - PLONE_VERSION=4.2
 - PLONE_VERSION=4.3
-- PLONE_VERSION=4.3 TEST_EGGS=plone.app.contenttypes
+- PLONE_VERSION=4.3 TEST_EGGS=buildout:test-eggs=plone.app.contenttypes
```

Comment:
I think we can remove this build and same some CO₂ emissions now that Plone 5 builds are passing.